dev/com.ibm.ws.concurrent.persistent_fat_configupdate_databasetaskstore/.classpath

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7">
```

Comment:
since the tests in this FAT only run on Java 8+, lets change this to be `JavaSE-1.8`